docs/platform-integration/essentials/email.md

```diff
@@ -28,7 +40,8 @@ Open the **AndroidManifest.xml** file under the **Properties** folder and add th
 
 # [iOS](#tab/ios)
 
-In iOS 9 and greater, Apple enforces what schemes an application can query for. To query if email is a valid target the `mailto` scheme must be specified in the  LSApplicationQueriesSchemes in your Info.plist file.
+<!-- TODO: Verify the wording on this. Something was wrong in the original article and I don't know iOS well enough -->
+In iOS 9 and greater, Apple enforces which schemes an application can query for. To query if **email** is a valid target, the `mailto` scheme must be specified for the `LSApplicationQueriesSchemes` scheme, in your _Info.plist_ file.
```

Comment:
I don't think that email needs to be in bold.

I'd say that the `mailto` scheme must be specified for the `LSApplicationQUeriesSchemes` key (rather than scheme).

This is the mechanism that gives your app permission to open the system email app.